src/har/request.cr

```diff
@@ -13,6 +13,8 @@ module HAR
   class Request
     include JSON::Serializable
 
+    SUPPORTED_MIME_BODY = %w[application/x-www-form-urlencoded multipart/form-data]
```

Comment:
`SUPPORTED_MIME_BODY` is way too general name. Also, Tuple could be used, instead of an Array.

```suggestion
    FORMDATA_MIME_VALUES = {"application/x-www-form-urlencoded", "multipart/form-data"}
```